classes/components/forms/context/MastheadForm.inc.php

```diff
@@ -50,6 +64,14 @@ public function __construct($action, $locales, $context, $imageUploadUrl) {
 				'size' => 'small',
 				'groupId' => 'publishing',
 				'value' => $context->getData('printIssn'),
+			]))
+			->addField(new FieldSelect('country', [
+				'groupId' => 'publishing',
```

Comment:
@NateWr, before I request a review... Is it ok to move the country field to the `PKPMastheadForm` and use the `identity` as `groupId` or at least create the `Publishing Details` group there? Just asking because OMP and OPS don't have this section.

Also, OMP has a "geographical location" field.